openfisca_us/variables/fcc/acp/is_acp_eligible.py

```diff
@@ -0,0 +1,21 @@
+from openfisca_us.model_api import *
+
+
+class is_acp_eligible(Variable):
+    value_type = bool
+    entity = SPMUnit
+    label = "Eligible for Affordable Connectivity Program"
+    documentation = "Eligible for Affordable Connectivity Program"
+    definition_period = YEAR
+
```

Comment:
```suggestion
reference = "47 U.S.C §1752(a)(6)"
```

Or the [link](http://uscode.house.gov/view.xhtml?req=granuleid:USC-prelim-title47-section1752&num=0&edition=prelim), though I don't think we can link to specific sections?

Reply:
Filed https://github.com/PolicyEngine/openfisca-us/issues/581 to come back to this once it's in the Cornell site